engine/src/main/java/org/terasology/logic/players/CameraClientSystem.java

```diff
@@ -65,7 +74,18 @@ public void resetCamera() {
     }
 
     @ReceiveEvent
-    public void resetCamera(ResetCameraEvent resetCameraEvent, EntityRef client, AutoMountCameraComponent autoMountCameraComponent, ClientComponent clientComponent) {
+    public void resetCamera(ResetCameraEvent resetCameraEvent, EntityRef entityRef, AutoMountCameraComponent autoMountCameraComponent, ClientComponent clientComponent) {
+        if (localPlayer.getClientEntity().equals(entityRef)) {
+            clientComponent.camera.destroy();
+            clientComponent.camera = EntityRef.NULL;
+            // this will trigger a ClientComponent change which will in turn trigger a remount
+            localPlayer.getClientEntity().saveComponent(clientComponent);
+        }
+    }
+
+    @Command(shortDescription = "Mounts the camera to the character", requiredPermission = PermissionManager.NO_PERMISSION)
```

Comment:
"Expiration date" could be simply adding `@deprecated` then we can just tolerate it while testing, but will be continuously reminded it shouldn't be a permanent thing :D